### ISSUE TITLE
BootLoop on v17.1

### ISSUE THREAD

**Issue by SamSensei** [topjohnwu/Magisk#579]
when selecting system apps for magisk hide , android system crashed and stuck on bootloop
note : uninstalled v16 and installed v17.1 magisk as previously after updating it gives bootloop
Device Specifications:-
Android 7.0
ROM -Xiaomi.eu MIUI v8.8.30 
device : Redmi Note 4X

**Comment by bmg002** [comment]
I'm having the same issue with my xiaomi mi 5s plus. I wonder if it is related to miui?
Although if I leave it long enough (an hour or so) out will usually boot up.

**Comment by moposx** [comment]
For my xiaomi 6x running an oreo 8.1 based miui, there is a problem of "Unknown baseband" that is seemed to be related to Magisk v17.1. After a clean reflashing and installed magisk 17.1, my phone suddenly rebooted with unknown baseband, then I restored stock boot, installed 16.7, and everythings worked fine.
PS: I have used v17.1 on  AEX days ago, also oreo 8.1 based, but there was no such problem. 
Very probably for miui's reason.
